spv/ancestors_json.go

```diff
@@ -9,8 +9,6 @@ import (
 
 // AncestryJSON is a spec at https://tsc.bitcoinassociation.net/standards/spv-envelope/ eventually.
 type AncestryJSON struct {
-	PaymentTx string         `json:"paymentTx,omitempty"`
-	Depth     uint64         `json:"depth,omitempty"`
 	Ancestors []AncestorJSON `json:"ancestors,omitempty"`
```

Comment:
do we still need this struct if there's only an array of ancestorJSON?

Reply:
I suppose we could! I didn't think of that.